lib/src/TailwindComponents/ButtonPureCss/button.module.css

```
@@ -0,0 +1,309 @@
+@import 'tailwindcss';
+
+/* The 'components' layer is used so that the styles can be overridden by Tailwind's 'utilities' layer */
+@layer components {
+  .root {
+    position: relative;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    cursor: pointer;
+    border-style: solid;
+
+    /* VARIABLES FROM THEME - DONT CHANGE WITH REACT PROPS */
+    border-width: var(--border-width-sm);
+    transition-property: all;
+    transition-duration: var(--duration-medium);
+    outline: transparent solid var(--border-width-md);
+    font-weight: var(--font-weight-bold);
+    border-radius: var(--radius-md);
+
+    &:focus {
+      outline-color: var(--outlineColorFocus);
+    }
+
+    &:hover {
+      background-color: var(--color-brand-30);
+      border-color: var(--color-brand-30);
+    }
+
+    &:active {
```

```suggestion
    &:active:not(:disabled):not([aria-disabled]) {
```

to avoid writing `disabled` styled for active/focus/hover under ⬇️